lib/menu.rb

```diff
@@ -0,0 +1,9 @@
+class Menu
+
+  attr_reader :dishes
+
+  def initialize
+    @dishes = {:Noodles=>3.0, :Rice=>2.5}
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator '=>'.<br>Space inside { missing.<br>Space inside } missing.